core/src/modules/ops/profileProperty.ts

```diff
@@ -2,8 +2,14 @@ import { parsePhoneNumberFromString, CountryCode } from "libphonenumber-js/max";
 import { plugin } from "../../modules/plugin";
 import isEmail from "../validators/isEmail";
 import isURL from "validator/lib/isURL";
+import { ProfileProperty } from "../../models/ProfileProperty";
+import { Property } from "../../models/Property";
+import { CLS } from "../../modules/cls";
+import { Op } from "sequelize";
 
 export namespace ProfilePropertyOps {
+  const defaultProfilePropertyProcessingDelay = 1000 * 60 * 5;
```

Comment:
I wonder if this default should be exposed as a setting.  I think it's ok to be only internal for now as the batch size is already configurable, which is the biggest component to any delays that occur